src/utils/trusts.ts

```diff
@@ -30,6 +30,13 @@ const checkEntityRequiresTrusts = (appData: ApplicationData | undefined): boolea
     containsTrusteeNatureOfControl(appData.beneficial_owners_corporate ?? []);
 };
 
+const checkEntityRequiresManageTrusts = (appData: ApplicationData | undefined): boolean => {
```

Comment:
return appData?.update?.review_trusts && appData.update.review_trusts.length > 0;
- if we wanted to do it all on one line

Reply:
Presumably we will have loaded trusts earlier. Only if the manage trusts flag is true.

Reply:
this complains about possibly undefined so current way came out as best solution